src/test/resources/fixtures/hmh/hmh-ri/released/caliperEventAssessmentStarted.json

```diff
@@ -1,35 +1,49 @@
 {
-  "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
-  "type":"AssessmentEvent",
-  "id":"urn:uuid:27734504-068d-4596-861c-2315be33a2a2",
-  "actor":{
-    "id":"urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
-    "type":"Person"
-  },
-  "action":"Started",
-  "object":{
-    "id":"urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
-    "type":"Assessment",
-    "name":"ReadingInventory",
-    "maxAttempts":1,
-    "maxSubmits":1,
-    "maxScore":25.0,
-    "version":"1.0"
-  },
-  "eventTime":"2016-11-15T10:15:00.000Z",
-  "edApp": "https://www.hmhco.com/ReadingInventory",
-  "membership":{
-    "id":"urn:uuid:randomGuidId",
-    "type":"Membership",
-    "organization":{
-      "id":"urn:uuid:ab570d90-3791-4048-a5ee-759657ddccef",
-      "type":"Organization",
-      "subOrganizationOf":{
-        "id":"urn:uuid:ea827de8-0ce9-4fe4-a28c-eb388bd9eb92",
-        "type":"Organization"
-      }
-    },
-    "roles":[ "Learner" ],
-    "status": "Active"
-  }
+	"@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+	"type": "AssessmentEvent",
+	"id": "urn:uuid:27734504-068d-4596-861c-2315be33a2a2",
+	"actor": {
+		"id": "urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+		"type": "Person"
+	},
+	"action": "Started",
+	"object": {
+		"id": "urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
+		"type": "Assessment",
+		"name": "ReadingInventory",
+		"maxAttempts": 1,
+		"maxSubmits": 1,
+		"maxScore": 25.0,
+		"version": "1.0"
+	},
+	"eventTime": "2016-11-15T10:15:00.000Z",
+	"edApp": {
+		"id": "https://www.hmhco.com/ReadingInventory",
+		"type": "SoftwareApplication",
+		"extensions": {
+			"applicationCode": "RI",
+			"disciplineCode": "ED18_RLA"
+		}
+	},
+	"membership": {
+		"id": "urn:uuid:randomGuidId",
+		"type": "Membership",
+		"organization": {
+			"id": "urn:uuid:sdfhs76h-2343-a087-34b3-df423kb2zjz8",
+			"type": "CourseSection",
+			"academicSession": "BOY 2019",
```

Comment:
Ed Analytics has only implemented the fields of parent type `Organization`, fields for subtypes such as `CourseSection` have not been implemented, consequently `academicSession` would not be picked up.

Reply:
Its okay if we did not pick these up until we added support. As far as we do not cause issues by pushing to the pipeline.

Reply:
AFAIK it wouldn't cause issues, it would simply be dropped and ignored by the pipeline